ocw/lib/db.py

```diff
@@ -34,18 +34,20 @@ def sync_csp_to_local_db(pc_instances, provider, namespace):
         if i.vault_namespace != namespace:
             raise ValueError('Instance {} does not belong to {}'.format(i, namespace))
 
-        logger.debug("Update/Create instance %s:%s @ %s", provider, i.instance_id, i.region)
-        if Instance.objects.filter(provider=i.provider, instance_id=i.instance_id).exists():
-            o = Instance.objects.get(provider=i.provider, instance_id=i.instance_id)
+        if Instance.objects.filter(provider=i.provider, instance_id=i.instance_id, vault_namespace=namespace).exists():
+            logger.debug("[%s] Update instance %s:%s", namespace, provider, i.instance_id)
+            o = Instance.objects.get(provider=i.provider, instance_id=i.instance_id, vault_namespace=namespace)
             if o.region != i.region:
-                logger.info("Instance %s:%s changed region from %s to %s",
-                            provider, i.instance_id, o.region, i.region)
+                logger.info("[%s] Instance %s:%s changed region from %s to %s",
+                            namespace, provider, i.instance_id, o.region, i.region)
                 o.region = i.region
             if o.state == StateChoice.DELETED:
-                logger.error("Update already DELETED instance %s:%s\n\t%s", provider, i.instance_id, i.csp_info)
+                logger.error("[{}] Update already DELETED instance {}:{} {}".format(
```

Comment:
Why do you need to move from the lazy formatting to ''.format(...)?

Reply:
reverted back